src/arch/xtensa/smp/init.c

```diff
@@ -94,13 +94,14 @@ int slave_core_init(struct sof *sof)
 	if (err < 0)
```

Comment:
Wat do you mean by " incremental trace points" can you enhance the message commit decription.

Reply:
Done.